### PR TITLE
feat: add ServerBindAddr to app examples, fixes #40

### LINKED ISSUE #40: Add new ServerBindAddr setting to all example's config files.
Examples need to be updated with latest SDK and config files updated to add ServerBindAddr setting

### PR DESCRIPTION
**Pull request by charles-knox-intel**
Signed-off-by: charles-knox-intel <44684517+charles-knox-intel@users.noreply.github.com>

Adds the `ServerBindAddr` changes from https://github.com/edgexfoundry/app-functions-sdk-go/pull/409 and https://github.com/edgexfoundry/app-functions-sdk-go/issues/405

### CODE REVIEW

**Review by lenny-goodell — CHANGES_REQUESTED** [review]
Missing update to the repo's go.mod file for latest SDK that adds ServerBindAddr

File: go.mod [hunk 1]
```diff
@@ -5,7 +5,7 @@ go 1.13
 require (
 	github.com/cloudevents/sdk-go v1.1.2
 	github.com/dghubble/sling v1.3.0
-	github.com/edgexfoundry/app-functions-sdk-go v1.0.1-dev.43
-	github.com/edgexfoundry/go-mod-core-contracts v0.1.57
-	github.com/stretchr/testify v1.5.1
+	github.com/edgexfoundry/app-functions-sdk-go v1.2.1-dev.9
+	github.com/edgexfoundry/go-mod-core-contracts v0.1.60
+	github.com/stretchr/testify v1.6.1
```
Comment (charles-knox-intel) [inline go.mod:10]: @lenny-intel , good catch, I have updated the dependencies. I was surprised that this repo was still on app-functions-sdk-go `v1.0.1-dev.43` still though. Any concerns about the upgrade?

Comment (lenny-goodell) [reply go.mod:10]: No concerns about bringing it up to latest. @rsdmike , this repo is a great candidate for DependABot.

**Review by lenny-goodell — APPROVED** [review]
LGTM

**Pull request merged**